ELY_Messages.txt

```diff
@@ -3,7 +3,7 @@
 
 00012            wildfly-elytron-asn1
 07004 - 07024    wildfly-elytron-asn1
-11001 - 11007    wildfly-elytron-audit
+11001 - 11010    wildfly-elytron-audit
```

Comment:
I don't think a reason to not merge this PR but we should not be attempting to extend any of the existing ranges, these existing ranges are already congested and TBH are an unmanageable mess making review error prone.

Instead we have plenty of unallocated blocks of 1,000 - if a module needs additional messages it should instead take one of the free blocks of 1,000.  There are plenty of these blocks available before we will start running out.

Reply:
Updated it to use the 12000-12002 range for the new messages